clients/venice-push-job/src/main/java/com/linkedin/venice/heartbeat/DefaultPushJobHeartbeatSender.java

```diff
@@ -25,7 +25,7 @@
 
 
 @NotThreadSafe
-class DefaultPushJobHeartbeatSender implements PushJobHeartbeatSender {
+public class DefaultPushJobHeartbeatSender implements PushJobHeartbeatSender {
```

Comment:
Please share if there is any concern here.

Reply:
This change is not necessary, since the factory calling this constructor is also in the new package.

Reply:
fixed.